src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -340,6 +348,7 @@ class FormPage extends React.Component {
             uploadFile={this.props.uploadFile}
             schema={schema}
             uiSchema={uiSchema}
+            getFormData={this.formData}
```

Comment:
Isn't `data` already providing full form data? I remember in the Array loop pattern, that it's only passing in the data for that page index, but this is the main `FormPage`. Or am I missing something?

Reply:
we discussed this offline. In this case looks like data can indeed be partial if its in an array.